### DISCUSSION TITLE
gantt diagram

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#14548] [How to]
Hi,

I have a hard time understanding the **gantt** project diagram, ie., based on my understanding it is not working.

1. I created a task with a start date 2021-04-12 and tentatively ending 2102-04-19.
![test](https://user-images.githubusercontent.com/10230144/114571623-1c9ca000-9c77-11eb-8ab6-f1cb2787c05d.png)

2. When looking a the gantt diagram on the project page, I see 1) a duration of 1000 days (in text) and 2) a duration of one day (in the figure)
![test-gantt](https://user-images.githubusercontent.com/10230144/114571874-481f8a80-9c77-11eb-9bcf-a60c8eadb829.png)

All data/testing is from gitdeveloper.yetiforce.com. Relevant data is marked in yellow by me.

**Comment by rioshaz** [comment]
i have the sme problem

**Comment by davide-alghi** [comment]
#15772
